### PR TITLE
Feature/event group list filters

### PR DESCRIPTION
**Pull request by sandorbanvitagroup**

### CODE REVIEW

File: src/app/events/event-components/event-groups-list/event-groups-list-action-data.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { ACTIONS_VIEW_OPTION } from '../../../_constants/actions';
+import { NavItem, NavItemRole } from '../../../_models/common';
+
+export const viewOptionsDefs: NavItem[] = [
+  {
+    role: NavItemRole.ACTION,
```
Comment (danielrus) [inline src/app/events/event-components/event-groups-list/event-groups-list-action-data.ts]: I think if you define role as NavItemRole.LINK and adjust the action constant (for example: ACTION_VIEW_OPTION.EVENTS to have value of '/events/list' ) it should work directly without additional work

File: src/app/_constants/actions.ts [hunk 2]
```diff
@@ -19,6 +19,12 @@ export const ACTIONS_BULK_EDIT = {
   QUARANTINE_DOCUMENTS: 'CREATE_QUARANTINE_ORDER_DOCUMENTS',
 };
 
+export const ACTIONS_VIEW_OPTION = {
```
Comment (danielrus) [inline src/app/_constants/actions.ts]: the name is too generic, maybe ACTIONS_EVENTS_VIEW_OPTIONS would be more suited (we have view options in multiple places, not only events page)

File: src/app/events/event-components/event-groups-list/event-groups-list-action-data.ts (hunk 1 above)
Comment (sandorbanvitagroup) [reply src/app/events/event-components/event-groups-list/event-groups-list-action-data.ts]: Resolved

File: src/app/_constants/actions.ts (hunk 2 above)
Comment (sandorbanvitagroup) [reply src/app/_constants/actions.ts]: Changed.

**Review by danielrus — APPROVED** [review]

**Review by adriandumavitagroup — APPROVED** [review]

**Pull request merged**